scripts/python/test_archive_repositories.py

```diff
@@ -1,17 +1,16 @@
-import sys
 import unittest
 from unittest.mock import patch
 
 from github import Github
 
-from scripts.python import archive_repositories
+import archive_repositories
 
 
 @patch.object(Github, "__new__")
 class TestArchiveRepositories(unittest.TestCase):
 
+    @patch("sys.argv", ["", "test"])
     def test_main_smoke_test(self, mock1):
-        sys.argv = ["", "test"]
```

Comment:
Setting this value was being carried over to other tests 😱 Which meant that other tests where if this value is set, the function ends up in an infinite loop, the unit tests never exited.

Using the @patch decorator means this value is only set during this test function 🔥 🚒